src/js/components/dashboard/summary/DashboardSummary.jsx

```diff
@@ -0,0 +1,78 @@
+/**
+  * DashboardSummary.jsx
+  * Created by Daniel Boos 11/12/19
+  */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import DashboardSummaryRow from './DashboardSummaryRow';
+
+const propTypes = {
+    file: PropTypes.string,
+    agency: PropTypes.string,
+    submissions: PropTypes.array
+};
+
+function getSummaryRows(submissions, file) {
+    const summaryRows = [];
+    let key = 0;
+    if (submissions.length === 0) {
+        summaryRows.push(<DashboardSummaryRow key={key} />);
+    }
+    else {
+        submissions.forEach((submission) => {
+            let period = '';
+            const fy = `${submission.fy}`.substring(2);
+            period = `FY ${fy}, Q${submission.quarter}`;
+            const subid = `${submission.submission_id}`;
+            summaryRows.push(<DashboardSummaryRow
+                key={key}
+                file={file}
+                period={period}
+                subID={subid}
+                submitter={submission.certifier} />
+            );
+            key += 1;
+        });
+    }
+    return summaryRows;
+}
+
+export default class DashboardSummary extends React.Component {
+    render() {
+        const summaryRows = getSummaryRows(this.props.submissions, this.props.file);
+
+        return (
+            <div>
+                <div className="dashboard-page__agency">
+                    <span className="dashboard-page__agency-icon">
+                        <FontAwesomeIcon icon="landmark" />
+                    </span>
+                    {this.props.agency}
+                </div>
+                <div className="table-responsive">
+                    <table className="dashboard-page__summary-table">
+                        <thead>
+                            <tr className="dashboard-page__summary-table-row">
+                                <th className="row-8" scope="col"><FontAwesomeIcon icon="file-alt" /></th>
```

Comment:
Could we combine the icon and the label into the same `<th>` element? This would also prevent empty `<td>` elements in the row component, but will require some added styling to get the spacing you want.